webextensions/api/windows.json

```diff
@@ -667,9 +667,9 @@
               "firefox": {
                 "version_added": "45",
                 "notes": [
-                  "'url' and 'tabId options can't both be set together.",
-                  "The returned 'Window' object contains the 'tabs' property only from version 52 onwards.",
-                  "From Firefox 86, the <code>focused: false</code> option is ignored.",
+                  "<code>url</code> and <code>tabId</code> options can't be set together.",
```

Comment:
We just recently switched to allowing Markdown formatting in notes, so let's use that now!

Reply:
@queengooborg Completed while resolving merge conflicts